ibis/backends/mysql/__init__.py

```diff
@@ -224,8 +224,15 @@ def get_schema(
         ).sql(self.dialect)
 
         with self.begin() as cur:
-            cur.execute(sge.Describe(this=table).sql(self.dialect))
-            result = cur.fetchall()
+            query = sge.Describe(this=table).sql(self.dialect)
```

Comment:
Let's avoid executing code under an open cursor if that code doesn't require the cursor to be open.